inference-engine/src/inference_engine/src/cnn_network_ngraph_impl.cpp

```diff
@@ -489,8 +489,8 @@ StatusCode CNNNetworkNGraphImpl::serialize(const std::string& xmlPath,
         ngraph::pass::Manager manager;
         manager.register_pass<ngraph::pass::Serialize>(xmlPath,
                                                        binPath,
-                                                       ngraph::pass::Serialize::Version::IR_V10,
-                                                       custom_opsets);
+                                                       custom_opsets,
+                                                       ngraph::pass::Serialize::Version::IR_V10);
```

Comment:
also - can we use information from the function itself?

Reply:
Discussed offline that if we read IR v11 in old API - it's always downgraded to IR v10